dockstore-client/src/main/java/io/dockstore/client/cli/Client.java

```diff
@@ -643,6 +644,7 @@ private void serverMetadata() {
             final Metadata metadata = ga4ghApi.metadataGet();
             final Gson gson = io.cwl.avro.CWL.getTypeSafeCWLToolDocument();
             out(gson.toJson(metadata));
+            out("server-url: " + serverUrl);
```

Comment:
Suggestion, your call -- I would make it more end-user friendly, e.g., "Dockstore server:", or "Dockstore url:". I don't think most users will know that `server-url` is a property in the config file, even they could probably deduce it.